HTTP POST - Watts to Thingspeak

```diff
@@ -0,0 +1,26 @@
+// Shelly GEN2 script: HTTP POST - Send energy meter watts to Thingspeak cloud
+
+// Settings
+let tsapikey = "YOURAPIKEY"; // Copy from Thingspeak.com > Channels > Channel > API Keys > Write API key
+let tsjsonurl = "https://api.thingspeak.com/update.json";
+
+// Assign watts to variable
+let status = Shelly.getComponentStatus("switch", 0);
+let watts = status.apower;
+print(watts);
+
+// Define timespan: minutes * 60 sec * 1000 milliseconds
+let interval = 5 * 60 * 1000;
+
+// Create JSON
+let tsjson = {
+	"api_key": tsapikey,
+	"field1": watts
+};
+
+// Set timer which send the HTTP POST
+Timer.set(
```

Comment:
I think the intention is to send data retrieved at regular intervals. The current script will only query the switch for the power consumption once and then regularly send the same number.

Can you rework that so access to component status is inside the timer?

Also the established convention is that there is a global CONFIG object holding script relevant configuration like access tokens or keys and endpoints. Can you change your script to have such an object?

Reply:
Good spot! Fixex the first part. 

Is there example how to CONFIG? Did not come across that in Shelly docs.